### PR TITLE
bump python version, 3.8 is not supported anymore

### PR DESCRIPTION
**Pull request by shcheklein**
Python 3.8 is not supported by DVC anymore.

### CODE REVIEW

File: recipe/meta.yaml
Comment (weiji14) [inline recipe/meta.yaml:1]: Can you also bump the build number from 0 to 1 at L16?

Comment (shcheklein) [reply recipe/meta.yaml:1]: done

**Comment by weiji14** [comment]
@conda-forge-admin, please rerender

**Review by weiji14 — APPROVED** [review]
Ok, should be good to merge once the tests pass. Thanks!

**Pull request merged**